src/views/v2/day/event/cost.php

```diff
@@ -9,7 +9,7 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.10.9
+ * @version TBD
```

Comment:
These should all be

```suggestion
 * @since 4.10.9
 * @version TBD
```

Reply:
Oh cool! didn't know we implemented the `@since` for templates. The `@version` has a key role when letting our users know, programatically, that they need to update their implementations (if any).

Is there any particular goodie with `@since` besides our internal docs?

Reply:
Specifies when it was introduced - that helps them as well as us.
We could but don't yet leverage it beyond that I believe?

Reply:
On the green team we use `@since` and `@version` to keep track of when the view was introduced and what changes each version were. It's been helpful for debugging and from the customer side as we can tell them when a specific view was introduced by looking at the code.

Reply:
Thanks!